drv/auxflash-server/src/main.rs

```diff
@@ -46,13 +46,23 @@ impl<'a> TlvcRead for SlotReader<'a> {
         dest: &mut [u8],
     ) -> Result<(), TlvcReadError<Self::Error>> {
         let addr: u32 = self.base + u32::try_from(offset).unwrap_lite();
-        self.qspi.read_memory(addr, dest);
+        // The QSPI driver was updated to return errors but this wants to
+        // be infallible
+        self.qspi.read_memory(addr, dest).unwrap_lite();
```

Comment:
Out of curiosity, did you try making `Self::Error` into `AuxFlashError` here (instead of `Infallible`)?

Reply:
I did, it still seemed to want an `Infallible` but I also didn't dig into it too closely.

Reply:
I was clearly not thinking when I tried this before and fixed it